src/main/java/com/google/cloud/tools/ide/login/AccountInfo.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.ide.login;
+
+import java.awt.Image;
+
+/**
+ * Represents information about a logged-in account. The information is intended for use by UI.
+ * Filled in and returned only by {@link GoogleLoginState#listAccounts()}.
+ *
+ * @see GoogleLoginState#listAccounts()
+ */
+public class AccountInfo {
+  String email;
+  String realName;
+  Image avatar;
+
+  AccountInfo(String email, String realName, Image avatar) {
+    this.email = email;
```

Comment:
Will annotate `name` and `avatar` with `@Nullable`. These have to be non-null under normal circumstances, but unlike `email`, I will allow them to be null if the library fails to retrieve them (i.e., not forcing it to be a login failure in this error situation).